From c4165b39531872b7b56d497c4ebd86b5d1d79800 Mon Sep 17 00:00:00 2001
From: Scott Little <scott.little@windriver.com>
Date: Wed, 25 Oct 2017 16:18:02 -0400
Subject: [PATCH] 
 Modify-error-code-of-bz1454699-fix-to-prevent-inactive-controller-reboot-loop

---
 SPECS/resource-agents.spec | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/SPECS/resource-agents.spec b/SPECS/resource-agents.spec
index 19580ef..2536cb7 100644
--- a/SPECS/resource-agents.spec
+++ b/SPECS/resource-agents.spec
@@ -252,6 +252,7 @@ Patch1116: ocf-shellfuncs_change_logtag.patch
 Patch1117: lvm_cleanup_refs_on_stop.patch
 Patch1118: ipaddr2_if_down.patch
 Patch1119: ipaddr2_ignore_lo_if_state.patch
+Patch1120: Modify-error-code-of-bz1454699-fix-to-prevent-inactive-controller-reboot-loop.patch
 
 Obsoletes:	heartbeat-resources <= %{version}
 Provides:	heartbeat-resources = %{version}
@@ -561,6 +562,7 @@ exit 1
 %patch1117 -p1
 %patch1118 -p1
 %patch1119 -p1
+%patch1120 -p1
 
 %build
 if [ ! -f configure ]; then
-- 
1.9.1

